x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ml/utils/MlStringsTests.java

```diff
@@ -22,6 +21,37 @@
 
 public class MlStringsTests extends ESTestCase {
 
+    public static final String[] someInvalidChars = {
```

Comment:
Static final immutable constants should have upper case names.

```suggestion
    public static final String[] SOME_INVALID_CHARS = {
```